### PR TITLE
Commit 3

### PR DESCRIPTION
**Pull request by SemBarner22**
Return of friends.

### CODE REVIEW

File: src/main/java/wp/cache/UserProfileData.java [hunk 1]
```diff
@@ -4,5 +4,5 @@
 public class UserProfileData {
     String team;
 
-
+    // Commit 3: return of friends
```
Comment (Garuda-1) [inline src/main/java/wp/cache/UserProfileData.java]: Why is this done?

File: src/main/java/wp/cache/UserProfileData.java [hunk 2]
```diff
@@ -4,5 +4,5 @@
 public class UserProfileData {
```
Comment (Garuda-1) [inline src/main/java/wp/cache/UserProfileData.java:4]: Rename

**Review by Garuda-1 — CHANGES_REQUESTED** [review]
Changed needed

File: src/main/java/wp/cache/UserProfileData.java (hunk 2 above)
Comment (SemBarner22) [reply src/main/java/wp/cache/UserProfileData.java:4]: No.

File: src/main/java/wp/cache/UserProfileData.java (hunk 1 above)
Comment (SemBarner22) [reply src/main/java/wp/cache/UserProfileData.java]: Will fix.

Comment (SemBarner22) [reply src/main/java/wp/cache/UserProfileData.java]: Fixed.

**Review by Garuda-1 — APPROVED** [review]
LGTM

**Pull request merged**